apps/omg_db/lib/omg_db/leveldb/core.ex

```diff
@@ -41,8 +41,11 @@ defmodule OMG.DB.LevelDBCore do
   @doc """
   Interprets the response from leveldb and returns a success-decorated result
   """
+  @spec decode_value({:ok, binary()} | :not_found, atom()) :: {:ok, term()} | {:error, term()}
   def decode_value(db_response, type) do
     case decode_response(type, db_response) do
+      # :not_found -> :not_found
```

Comment:
these changes have been different to later this week, because we need some more extensive e2e testing thats not yet ready!